setup.py

```diff
@@ -81,7 +81,7 @@ def open_local(paths, mode="r", encoding="utf8"):
     '; sys_platform != "win32" ' 'and implementation_name == "cpython"'
 )
 ujson = "ujson>=1.35" + env_dependency
-uvloop = "uvloop>=0.5.3,<0.15.0" + env_dependency
+uvloop = "uvloop>=0.5.3,!=0.15.0,!=0.15.1" + env_dependency
```

Comment:
```suggestion
uvloop = "uvloop>=0.5.3" + env_dependency
```

Reply:
This pinning goes away completely with https://github.com/MagicStack/uvloop/issues/411